py/README.rst

```diff
@@ -12,7 +12,7 @@ The `selenium` package is used automate web browser interaction from Python.
 +-----------+-----------------------------------------------------------------------------------+
 | **Home**: | http://www.seleniumhq.org                                                         |
 +-----------+-----------------------------------------------------------------------------------+
-| **Docs**: | `selenium package API <http://selenium.googlecode.com/git/docs/api/py/api.html>`_ |
+| **Docs**: | https://selenium-python.readthedocs.io/ |
```

Comment:
no, we don't own / maintain that site. Should be:
http://seleniumhq.github.io/selenium/docs/api/py/api.html